apps/glusterfs/models.go

```diff
@@ -25,6 +25,25 @@ import (
 	"sort"
 )
 
+//Topology
+
+type Node struct {
+	NodeInfo NodeInfoResponse `json:"nodeinfo"`
+}
+
+type Nodes struct {
+	Nodes []Node
+}
+
+type Cluster struct {
+	Id       string `json:"id"`
+	NodeList Nodes  `json:"nodes"`
+}
+
+type Topo struct {
+	ClusterList []Cluster `json:"clusters"`
+}
+
```

Comment:
@lpabon 'cluster' or 'id of cluster' is part of nodeinfo response. Thats why its available in nodeinfo field. Considering the fact that its just one ID field, I feel its fine, so that we return the node information intact for some one to make use of it, if needed. Thats my thought. Please revert if you have any comments. Regarding 'models.go' for cli, I thought about it before placing it in server models.  but do u think at any stage we may have to create a route for topology in server side? in that case, this may be a better place. What do you think?